### PR TITLE
feat(quota): move quota tab - [do not merge yet]

### PR DESCRIPTION
**Pull request by frenautvh**
# do not merge

pay as you go upcoming changes: moving quota tab

### CODE REVIEW

File: client/app/cloud/project/compute/infrastructure/virtualMachine/addEdit/cloud-project-compute-infrastructure-virtualMachine-addEdit.html [hunk 1]
```diff
@@ -1263,12 +1263,22 @@ <h4 data-translate="cpcivm_addedit_advanced_options_description"></h4>
                                translate-compile
                                data-ng-if="VmAddEditCtrl.getMaximumInstanceCreationCount() > 1"
                                data-translate="cpcivm_addedit_advanced_options_multiple_creation_infos"
-                               data-translate-values="{ max : VmAddEditCtrl.getMaximumInstanceCreationCount(), type : VmAddEditCtrl.vmInEdition.flavor.name, region : VmAddEditCtrl.vmInEdition.region }"></p>
+                               data-translate-values="{
+                                   max : VmAddEditCtrl.getMaximumInstanceCreationCount(),
```
Comment (antleblanc) [inline client/app/cloud/project/compute/infrastructure/virtualMachine/addEdit/cloud-project-compute-infrastructure-virtualMachine-addEdit.html]: no `key-spacing` (see `sref`:))

File: client/app/cloud/project/compute/cloud-project-compute.html [hunk 2]
```diff
@@ -23,6 +23,8 @@
                          data-text="'cpc_tab_loadbalancer'|translate"
                          data-state="iaas.pci-project.compute.loadbalancer"
                          data-state-params="{ serviceName: CloudProjectComputeCtrl.serviceName }"></cui-tab>
+                <cui-tab data-text="'cpb_tab_quota' | translate"
```
Comment (marie-j) [inline client/app/cloud/project/compute/cloud-project-compute.html]: `::`

File: client/app/cloud/project/compute/quota/cloud-project-compute-quota.html [hunk 3]
```diff
@@ -1,39 +1,39 @@
-<div class="cloud-billing-quota cui-page__content">
+<div class="cloud-compute-quota cui-page__content">
 
     <!-- INIT LOADER -->
-    <div class="col-xs-12 text-center" data-ng-if="CloudProjectBillingQuotaCtrl.loader.quota">
+    <div class="col-xs-12 text-center" data-ng-if="CloudProjectComputeQuotaCtrl.loader.quota">
```
Comment (marie-j) [inline client/app/cloud/project/compute/quota/cloud-project-compute-quota.html]: I'm not sure the `.col-xs-12` is necessary here

File: client/app/cloud/project/compute/quota/cloud-project-compute-quota.html [hunk 4]
```diff
@@ -61,14 +61,14 @@
         <!-- QUOTA TABLE -->
         <strong>{{ :: 'cpb_quota_current_limit' | translate }}</strong>
 
-        <oui-datagrid data-rows="CloudProjectBillingQuotaCtrl.datas.quota">
+        <oui-datagrid data-rows="CloudProjectComputeQuotaCtrl.datas.quota">
             <oui-column data-title="'cloud_common_region' | translate" data-property="region" data-sortable="asc">
```
Comment (marie-j) [inline client/app/cloud/project/compute/quota/cloud-project-compute-quota.html]: `::`

**Pull request merged**